host/libhackrf/src/hackrf.h

```diff
@@ -458,6 +458,10 @@ extern ADDAPI int ADDCALL hackrf_start_rx_sweep(
 	hackrf_sample_block_cb_fn callback,
 	void* rx_ctx);
 
+extern ADDAPI size_t ADDCALL hackrf_get_transfer_buffer_size(void);
+
+extern ADDAPI uint32_t ADDCALL hackrf_get_transfer_queue_depth(void);
+
```

Comment:
Should we make these take a `hackrf_device*`? There may be scenarios where someone's using e.g. one HackRF for RX and one for TX, with different latency/throughput considerations.